assets/css/input.css

```diff
@@ -123,7 +123,7 @@ section {
 }
 
 .hello__nav {
-    @apply hidden m-auto md:flex justify-evenly align-top w-10/12 max-w-6xl h-full text-2xl font-extrabold;
+    @apply hidden m-auto md:flex justify-evenly align-top /* w-10/12 */ /* max-w-6xl */ h-full text-2xl font-extrabold;
```

Comment:
Tohle zakomentovávání bych nedělal. Buď úplně smazat nebo nechat. Akorát to předáme dalšímu představenstvu a budou zvedat obočí co to má dělat.

Reply:
fair point, tam zůstalo z developmentu